dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props,~../cnf/resources/bnd/transform.props
+
+Bundle-Name: JetBrains Kotlin Jakarta
+Bundle-SymbolicName: io.openliberty.org.jetbrains.kotlin
+Bundle-Description: Liberty JetBrains Kotlin, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  !android.net.http,\
+  javax.net,javax.net.ssl
+
+Export-Package: \
+  kotlin;version="1.0.0",\
+  kotlin.*;version="1.0.0"
+
+instrument.disabled: true
+
+-fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore
+
+-buildpath: \
+  org.jetbrains:annotations;version=13.0,\
```

Comment:
13 is a really old version and you're not exporting any packages from here ... are you sure you need it? If you do need it then actually I think it should be a separate bundle.